whiteBoarding/splitMiddleStr/keg-splitMiddleStr.js

```diff
@@ -0,0 +1,13 @@
+// Here's a start...10-28-18
+
+str1 = abcdefghijklmnopqrstuvxyz
+str2 = 123456789
+
+function(str1, str2){
+  let l2 = str2.length
+  let l1 = str1.length
+  let a = str1.splice(0, x)
```

Comment:
```/Users/edwardweymouth/code/practice/keg-middle.js:7
  let a = str1.splice(0, x)
                         ^

ReferenceError: x is not defined
```

Reply:
changed variable names in only one place...